packages/suite-desktop/scripts/build.js

```diff
@@ -29,31 +30,41 @@ const mockPlugin = {
     },
 };
 
-console.log('[Electron Build] Starting...');
-const hrstart = process.hrtime();
-build({
-    entryPoints: ['app.ts', 'preload.ts', ...modules].map(f => path.join(electronSource, f)),
-    platform: 'node',
-    bundle: true,
-    target: 'node14.16.0', // Electron 12
-    external: Object.keys({
-        ...pkg.dependencies,
-        ...pkg.devDependencies,
-    }),
-    tsconfig: path.join(electronSource, 'tsconfig.json'),
-    sourcemap: isDev,
-    minify: !isDev,
-    outdir: path.join(__dirname, '..', 'dist'),
-    define: {
-        'process.env.COMMITHASH': JSON.stringify(gitRevision),
-    },
-    plugins: useMocks ? [mockPlugin] : [],
-})
-    .then(() => {
-        const hrend = process.hrtime(hrstart);
-        console.log(
-            '[Electron Build] Finished in %dms',
-            (hrend[1] / 1000000 + hrend[0] * 1000).toFixed(1),
-        );
-    })
-    .catch(() => process.exit(1));
```

Comment:
Why are we removing this? I'd expect we want the process to exit with non-zero status on un-handled error.

Reply:
If it fails, the process will exit anyway.

Reply:
Aha, I didn't know this finally [got released](https://developer.ibm.com/languages/node-js/blogs/nodejs-15-release-blog/) with node v15. .) Just to make sure: are we using v15 or higher to run this script in CI, do we? .D (If yes, feel free to resolve this.)